### PR TITLE
improve docs user story for members to join orgs

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] **no todos or tks remain for this PR**
- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

* Initially, Raj brought up the need to have docs specifically for members to be "invited" (or as close to "invited" as we can make them feel).
* While we don't have an explicit invite feature yet, we do have live improvements to a member's first sign in that haven't been documented.
* When I reviewed the docs, they are out of sync with the product in the sense that the current docs focus solely on the first-person-in-an-org to use Semgrep and didn't accommodate "team members". **With this update, orgs and team members are better accommodated**.
* I opted to rewrite the doc focusing on **guiding** both user paths, with **steps** rather than conceptual info that can be read elsewhere.

### CODE REVIEW

**Comment by sachishah** [comment]
@s-santillan , for Joining an Org, should we link out to the SSO instructions?

For scanning, we're making several improvements to zero-config scanning (goal is to get it to a Public Beta state by the end of the week) as well as the recommended path for new users — should we hold off on the changes here until the product updates are live over the next two weeks?

File: docs/pricing-and-billing.mdx
Comment (armchairlinguist) [inline docs/pricing-and-billing.mdx:31]: Can we turn the inclusions here into a bullet list? I _think_ that's allowed within a definition list, and it would be better now that there are three:

The Team tier includes:
* Semgrep Code and Semgrep Supply Chain (SSC), which enable users to scan both first-party and third-party open source dependencies. 
* <strong>Cross-function (intrafile)</strong> and <strong>cross-file (interfile) analysis</strong> through <a href="/semgrep-code/semgrep-pro-engine-intro/">Semgrep Pro Engine</a>.
* Semgrep Cloud Platform, a web app enabling users to manage users, organizations, repositories, and scans.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:103]:
```suggestion
- Your admin or inviter should have an existing organization (org) in Semgrep that is connected to an org in either GitHub or GitLab.

```

Comment (armchairlinguist) [reply docs/semgrep-cloud-platform/getting-started.md:103]: "Inviter" is a bit weird here (funny since we use "invitee" a lot, but I don't hear "inviter" much). I might go with "Your admin or the person inviting you to SCP".

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:107]:
```suggestion
For **members**, perform the following steps to join the org:
```
I'm open to other alternative phrasing here, but the current "to join an organization...to be added to the org" feels heavy and redundant.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:110]:
```suggestion
2. Check that the organization under the **Join an organization** button is the correct org to join. If there is more than one org, select the org to join.

```

Comment (armchairlinguist) [reply docs/semgrep-cloud-platform/getting-started.md:110]: I don't think this sentence meets any of the exception criteria in the Google style guide for small numbers, but let me know if I'm mistaken!

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:132]: I think this is usually true, but is it always true?

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:136]:
```suggestion
1. Log in to Semgrep:

```

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:148]:
```suggestion
Many improvements to the Semgrep Cloud Platform experience only work with up-to-date Semgrep CLI versions. For this reason, Semgrep Cloud Platform only supports the 10 most recent minor versions of Semgrep CLI. For example, if the latest release was 0.114.0, all versions greater than 0.104.0 are supported, while earlier versions, such as 0.103.0, can be deprecated or can result in failures.

```

File: src/components/procedure/_platform-detect-ghrepos.md
Comment (armchairlinguist) [inline src/components/procedure/_platform-detect-ghrepos.md:4]: I don't think this set of steps will work for organizations as you need to select your org first. What would be the best way for us to handle that in this snippet?

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [reply docs/semgrep-cloud-platform/getting-started.md:136]: https://developers.google.com/style/word-list#login

I'll drop a comment on our internal example that uses `login` too :)

**Comment by armchairlinguist** [comment]
@sachishah I wouldn't recommend that we try to lump this in with changes for cloud scanning. It will delay us from making this update, which is an important one, and make it harder to review the changes.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:137]: Nice catch. This is one of those things I forget about when editing my work! Thank you.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:133]: There is one case I haven't verified, which is a case when someone pulls an org repo (say semgrep) then uploads findings from semgrep to the org - what happens when another user does that? I'll ask the cloud folks.

Comment (armchairlinguist) [reply docs/semgrep-cloud-platform/getting-started.md:133]: I think it might overwrite in that case, potentially. My observation is it's usually `org_name/repo_name` from CI and just `repo_name` locally. 

I was also thinking about if someone goes and reads about environment variables and sets the name, but that's pretty unlikely when they're just getting started.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:111]: I thought it fell under "**Technical quantities**, such as amounts of memory, amounts of disk space, numbers of queries, or usage limits" which is e

Comment (armchairlinguist) [reply docs/semgrep-cloud-platform/getting-started.md:111]: Ah I see. I don't feel strongly about it, so feel free to ignore.

File: src/components/procedure/_platform-detect-ghrepos.md
Comment (s-santillan) [reply src/components/procedure/_platform-detect-ghrepos.md]: Nice catch, I created separate steps for personal vs org users :)

File: docs/pricing-and-billing.mdx
Comment (s-santillan) [reply docs/pricing-and-billing.mdx]: Yep, resolved in latest :)

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:133]: Fixed in latest, I'll add a screenshot in the main comment body.

**Comment by s-santillan** [comment]
@sachishah I have added some links to SSO configuration to ensure they don't fall off the user path.



@armchairlinguist I added a section explaining how project slugs behave between local and remote scans

Snag_c15dd4

Thank you both for your notes, I feel that we have polished this doc quite a bit! Please approve if all good.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:104]: We may want to avoid use of the word "invite" here because folks are not actually invited. Maybe just "Your admin" should be enough?

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:112]: Should DESCRIPTION still be here?

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:146]: A lot of CLIs don't allow a direct click on this link - but not sure if that distinction really matters?

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:153]:
```suggestion
#### Project separation between local and remote repositories
```
Projects sounds a bit odd to me. If we do change this, the link above would need to be changed too.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:155]:
```suggestion
The Project slug for a remote repository takes the form `account-name/repository-name`. The Project slug for a local repository takes the form `repository-name`. Refer to the following screenshot for an example of both remote and local Projects in a single personal account.
```
we could add "by default", but mostly I just don't think it makes sense to say "always" even if it's just in a getting started page, since it's a configurable value.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:157]:
```suggestion
* **For personal accounts:** A local repository scan does not overwrite the findings records of its remote counterpart. They are two separate Projects. Personal accounts only have one team member or user: you.
```

File: docs/semgrep-cloud-platform/getting-started.md
Comment (armchairlinguist) [inline docs/semgrep-cloud-platform/getting-started.md:158]:
```suggestion
* For **organization accounts**: A local repository scan does not overwrite the findings records of its remote counterpart, but if two members both send local repository findings, their records can overwrite each other's findings. This is because org accounts can have more than one team member, but all local scans are sent to the same Project slug.
```
"their records can overwrite each other's findings" sounds weird here but I can't think of a great replacement. If you can, though, I'm for using it :)

**Review by armchairlinguist — APPROVED** [review]
This is a huge improvement for sure, thank you for all the hard work, @s-santillan ! I still had a few comments, mainly about the use of always/never about project names, but I leave it to your discretion.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md]: You're right, it's awkward. 

Alternative: if two members both send local repository findings, one set of results may overwrite the other unintentionally.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:146]: Click or copy. In the latest client, I was amazed that my WSL actually automatically opened my windows program. I am truly living in the future, 2 OS in 1. Cygwin could never.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:104]: This is so tricky. I used the term "invite" because that's the language people used to roughly and casually describe their goal, and is the gap we're trying to address because we don't have the functionality in the product per se.

Alternatively, I could remove the above usage of "invite", then explicitly state that Semgrep does not give out email invites, and then instruct admins to tell their team members to sign in after the admin has made an org.

What do you think?

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md:112]: Nice catch. Fixed in latest.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md]: Makes sense. I think removing the word "always" or "never" solves it :) Thank you.

File: docs/semgrep-cloud-platform/getting-started.md
Comment (s-santillan) [reply docs/semgrep-cloud-platform/getting-started.md]: Update: Yeah, I went in this direction :)

**Pull request merged**
